tools/code-analyzer/src/commands/analyzer/analyzer-major-minor.ts

```diff
@@ -62,14 +25,17 @@ const program = new Command()
 		const tmpRepoPath = await cloneRepo( source );
 		Logger.endTask();
 
-		const version = await getPluginData(
+		Logger.startTask( `Getting version from ${ pathToMainFile }` );
+		const pluginData = await getPluginData(
```

Comment:
So just to remind you, if you throw instead in `getPluginData` you can do your try catch here and call Logger.error in that case.